earthmover/nodes/destination.py

```diff
@@ -57,15 +58,17 @@ def __init__(self, *args, **kwargs):
         )
 
         #
-        try:
-            with open(self.template, 'r', encoding='utf-8') as fp:
-                template_string = fp.read()
-
-        except Exception as err:
-            self.error_handler.throw(
-                f"`template` file {self.template} cannot be opened ({err})"
-            )
-            raise
+        if self.template:
```

Comment:
good point, I've moved it into `execute()`

Reply:
(I kept the if test instead of using the default on `assert_get_key()` because if `template` is defined, it's a path to a template file, not a Jinja template string.)